docs/conf.py

```diff
@@ -123,14 +123,6 @@
     "_api/airflow/providers/cncf/index.rst",
     # Packages without operators
     "_api/airflow/providers/sendgrid",
-    # Other API rst files.
-    "_api/endpoints/index.rst",
-    "_api/endpoints/*/index.rst",
-    "_api/exceptions/index.rst",
-    "_api/parameters/index.rst",
-    "_api/schemas/index.rst",
-    "_api/schemas/*/index.rst",
-    "_api/security/index.rst",
```

Comment:
What are these changes for?

Reply:
I guess they might have been failing because of missing __init__.py file in api_connexion

Reply:
Yes, the version of autoapi we uses (1.0.0) doesn't support namespace packages, and so without `airflow/api_connextion/__init__.py` treats "airflow/api_connextion/schema/*.py" as a top-level `schema` package.

`_api/` is the output folder where autoapi writes its files to.